alpha/scripts/utils/getZoomToken.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+if($argc != 2)
+{
+	echo "Arguments missing.\n\n";
+	echo "Usage: php getZoomToken.php {vendor account id}\n";
+	exit;
+}
+
+$vendorAccountId = $argv[1];
+
+require_once(__DIR__ . '/../bootstrap.php');
+
+$zoomIntegration = ZoomHelper::getZoomIntegrationByAccountId($vendorAccountId);
```

Comment:
If not object return - let's print appropriate message and stop the scipt.

Reply:
VerifyZoomIntegration take care of that